app/controllers/hostgroups_controller.rb

```diff
@@ -74,15 +72,6 @@ def destroy
     end
   end
 
-  def current_parameters
```

Comment:
This will be removed in: https://github.com/theforeman/foreman/pull/3011 so I didn't want to change it here just to remove it when rebasing.